bot.py

```diff
@@ -187,30 +187,43 @@ def commandMessage(response):
 def adminMessage(response):
     # Repeats above except for admin commands
     if response["text"][:1] == "$":
-        if response["user"] in ADMIN:
+        # Checks if message is longer than "$"
+        if len(response["text"]) > 1:
             args = response["text"].split()
             com_text = args[0][1:].lower()
             args.pop(0)
-            # Special case for calendar requiring unique arguments
-            if com_text == "calendar":
-                if GOOGLECAL:
-                    rmsg(response, scripts.calendar.calendar(args, calendar_obj))  # noqa: 501
-                    return True
-            l = []
-            l.append(response)
-            l.append(args)
-            l.append(sc)
-            l.append(rmsg)
+            # Checks if pattern differs from admin commands
+            # by containing digits or another "$" character
+            com_pattern = (re.compile("[0-9]"), re.compile("[$]"))
             try:
-                f = getattr(commands[com_text], com_text)
-                f(*l)
-                return True
+                if com_pattern[0].search(com_text) or com_pattern[1].search(com_text)
+                return False
             except:
-                rmsg(response, ["You seem to have used a function that doesnt exist, or used it incorrectly. See `!help` for a list of functions and parameters"])  # noqa: 501
```

Comment:
I'm not sure but I don't believe Python supports actually having nothing inside the except, so you should probably add a `continue` statement here